azurerm/provider.go

```diff
@@ -104,11 +104,12 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_lb_rule":                 resourceArmLoadBalancerRule(),
 			"azurerm_local_network_gateway":   resourceArmLocalNetworkGateway(),
 
-			"azurerm_managed_disk":           resourceArmManagedDisk(),
-			"azurerm_network_interface":      resourceArmNetworkInterface(),
-			"azurerm_network_security_group": resourceArmNetworkSecurityGroup(),
-			"azurerm_network_security_rule":  resourceArmNetworkSecurityRule(),
-			"azurerm_public_ip":              resourceArmPublicIp(),
+			"azurerm_managed_disk":                  resourceArmManagedDisk(),
+			"azurerm_network_interface":             resourceArmNetworkInterface(),
+			"azurerm_network_security_group":        resourceArmNetworkSecurityGroup(),
+			"azurerm_network_security_rule":         resourceArmNetworkSecurityRule(),
+			"azurerm_operational_insight_workspace": resourceArmOperationalInsightWorkspaceService(),
```

Comment:
I'm a little confused around the naming of this resource, is the latest product name for this "Operational Insights" or "Log Analytics"?

Reply:
I also confused about that. I discuss with the production team.  In conclusion, they'd love to go "Log Analytics" for terraform.tf. However, SDK and REST-API is "Operational Insights".  I edit my code to fit to it.